src/js/node/child_process.ts

```diff
@@ -1359,9 +1360,17 @@ function nodeToBun(item, index) {
   if (typeof item === "number") {
     return item;
   }
+  if (item instanceof StreamModule.Readable) {
+    if (Object.hasOwn(item, "fd") && typeof item.fd === "number" && item.fd >= 0 && item.fd <= 2) return item.fd;
+    throw new Error(`TODO: stream.Readable stdio @ ${index}`);
```

Comment:
can we just silently ignore for now and have a debug warning? this might break something which wasn't necessarily previously broken

Reply:
this path was already throwing with `Invalid stdio option: "[object ReadStream]"`. these throws make the error more descriptive